### PR TITLE
PSVAMB-4967 PSVAMB-4944 KMS Email Notifications update

### PR DESCRIPTION
**Pull request by yusitnikov**
1. Rename notifications to make the names unique
2. PSVAMB-4967 - change the wording and add relevant links to KMS instance pages
3. PSVAMB-4944 - allow using an empty string as "instance_id" value

### CODE REVIEW

File: deployment/updates/scripts/xml/notifications/2019_01_15_update_unique_mediaspace_notifications_templates.template.xml [hunk 1]
```diff
@@ -0,0 +1,586 @@
...
+                    <!-- New parameters -->
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>category_id</key>
+                        <description>Category ID</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>$scope->getObject()->getCategoryId()</code>
+                        </value>
+                    </item>
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>user_id</key>
```
Comment (hilak) [inline deployment/updates/scripts/xml/notifications/2019_01_15_update_unique_mediaspace_notifications_templates.template.xml]: @yusitnikov I'd name the parameter something like 'executing_user_id' or 'ks_user_id'.

File: deployment/updates/scripts/xml/notifications/2019_01_15_update_unique_mediaspace_notifications_templates.template.xml [hunk 2]
```diff
@@ -0,0 +1,586 @@
...
+                        <description>User ID who executed this action</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>($ks = $scope->getKs()) &amp;&amp; ($kUser = $ks->getKuser()) ? $kUser->getPuserId() : ''</code>
```
Comment (hilak) [inline deployment/updates/scripts/xml/notifications/2019_01_15_update_unique_mediaspace_notifications_templates.template.xml]: @yusitnikov have you tested that this works? I seem to recall that in order for the && relationship to be enforced it should look like this:
(($ks = $scope->getKs()) &amp;&amp; ($kUser = $ks->getKuser())) ? .....

Comment (yusitnikov) [reply deployment/updates/scripts/xml/notifications/2019_01_15_update_unique_mediaspace_notifications_templates.template.xml]: @hilak , yes I tested it, but I added extra braces now anyway to keep the code more obvious

File: deployment/updates/scripts/xml/notifications/2019_01_15_update_unique_mediaspace_notifications_templates.template.xml (hunk 1 above)
Comment (yusitnikov) [reply deployment/updates/scripts/xml/notifications/2019_01_15_update_unique_mediaspace_notifications_templates.template.xml]: Done.

**Comment by kaltura-hooks** [comment]
Hi @yusitnikov,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

**Comment by yossipapi** [comment]
What about the saas pull request? And also I am missing the deployment script.

**Comment by yusitnikov** [comment]
@yossipapi , done.
`php /opt/kaltura/app/deployment/updates/scripts/2019_01_15_update_unique_mediaspace_notifications_templates.php`

File: deployment/updates/scripts/2019_01_15_update_unique_mediaspace_notifications_templates.php [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+ * @package deployment
+ */
+
+define('DEPLOYMENT_DIR', realpath(__DIR__ . '/../..'));
+
+require_once (DEPLOYMENT_DIR . '/bootstrap.php');
+
+$scriptPath = realpath(DEPLOYMENT_DIR . '/../tests/standAloneClient/exec.php');
+
+$templateUpdateXmlPath = realpath(DEPLOYMENT_DIR . '/updates/scripts/xml/notifications/2019_01_15_update_unique_mediaspace_notifications_templates.template.xml');
```
Comment (yossipapi) [inline deployment/updates/scripts/2019_01_15_update_unique_mediaspace_notifications_templates.php]: You need to check the existence of the actual file here not the template one please fix.
2019_01_15_update_unique_mediaspace_notifications_templates.template.xml ==> 2019_01_15_update_unique_mediaspace_notifications_templates.xml

Also, please add release notes for this.

Comment (yusitnikov) [reply deployment/updates/scripts/2019_01_15_update_unique_mediaspace_notifications_templates.php]: @yossipapi , done: pointed to the right file, updated from the master, added release notes

**Pull request merged**